### DISCUSSION TITLE
Encountered unknown type for Mysql: enum

### DISCUSSION THREAD

**Discussion by IoTResearchLLC** [diesel-rs/diesel#3378] [Q&A]
diesel print-schema
Encountered unknown type for Mysql: enum

I am using enum types in many tables. Is there a way to fix this or a workaround?

TIA

**Comment by weiznich** [comment]
It's not possible to answer this without knowing the relevant context. Please at at least the following details to your questions:

* Information about your diesel version
* Information about your database system (which one is used + what version)
* Relevant parts of your schema
* What's the expected output there. It's unclear what `enum` should even map to.

**Comment by adwhit** [comment]
I opened an issue - https://github.com/diesel-rs/diesel/issues/3386
